src/Bicep.Core/Syntax/NoneTypeLiteralSyntax.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.Syntax
+{
+    public class NoneTypeLiteralSyntax : TypeSyntax
```

Comment:
Do you actually need this?

Reply:
I thought we need it, but maybe we don't :) here in this commit (https://github.com/Azure/bicep/pull/13900/commits/66612746d32bf17fcb0cfcac609c9ee218e54760) I removed it 👍